src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -111,7 +112,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
             string runtimeMetadataVersion = null;
             bool errorEndLocation = false;
             bool reportAnalyzer = false;
-            string instrument = "";
+            ArrayBuilder<InstrumentationKind> instrument = ArrayBuilder<InstrumentationKind>.GetInstance();
```

Comment:
In the past John said there would be an unbounded number of such kinds, but I still don't understand the concept of different instrumentation kinds.
Would you have an example of two different "kinds" (to clarify the concept) which can be simultaneously be instrumented (to justify why we need an array of them).